README.md

```diff
@@ -8,7 +8,7 @@ What `swift-log` provides today can be found in the [API docs][api-docs]. At thi
 
 ---
 
-**NOTE**: You will need Xcode 10.2 beta 4 or a recent [Swift 5.0 development snapshot](http://swift.org/download/#swift-50-development) to try out `swift-log`.
+**NOTE**: You will need Xcode 10.2 beta 4, [Xcode 10.2](https://itunes.apple.com/us/app/xcode/id497799835) or a recent [Swift 5.0 development snapshot](http://swift.org/download/#swift-50-development) to try out `swift-log`.
```

Comment:
oh thanks. If you don't mind you could actually just say that you need Xcode 10.2 or Swift 5.0 . Don't think the dev snapshots or betas are still relevant and we won't unit test them anyway so I think they can be removed

Reply:
Great call, fixed.